content/terraform/v1.12.x/data/language-nav-data.json

```diff
@@ -1173,5 +1088,9 @@
       }
     ]
   },
+  {
```

Comment:
Hmmm top-level you think? It does kind of feel good here but I could also see it living in "configuration blocks. But I'm honestly down with either, just more curious if you'd considered the alternative.